presto-native-execution/presto_cpp/presto_protocol/tests/DeleteTest.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <gtest/gtest.h>
+
+#include "presto_cpp/main/common/tests/test_json.h"
+
+using namespace facebook::presto::protocol;
+
+using namespace ::testing;
+
+class DeleteTest : public ::testing::Test {};
+
+TEST_F(DeleteTest, TestJsonRoundtrip) {
```

Comment:
nit : naming convention... Individual unit tests do not use 'test' in names. test names start in lower case... so just jsonRoundTrip is fine here.